src/Avatar.tsx

```diff
@@ -8,6 +8,10 @@ type StyledAvatarProps = {
   size?: number
   /** Sets the shape of the avatar to a square if true. If false, the avatar will be circular. */
   square?: boolean
+  /** URL of the avatar image. */
+  src: string
+  /** Provide an alt text in case the Avatar is used without the user's name next to it. */
```

Comment:
[Minor] I could see this being misunderstood as "always provide alt text just in case the avatar isn't used next to a username." I think we could reword it to make a little more clear:
```suggestion
  /** Provide alt text when the Avatar is used without the user's name next to it. */
```